### ISSUE TITLE
Information stream is not passed to output

### ISSUE THREAD

**Issue by ajansveld** [PaulHigin/PSThreadJob#4]
This does not generate output (substituting Start-Job does):
```
Start-ThreadJob {Write-Information "foo" -InformationAction Continue} | Receive-Job -Wait -AutoRemoveJob
```

Is this by design?

**Comment by ajansveld** [comment]
And perhaps related to this, using Write-Host in the script block sends the data to the output instead of the Information stream.

**Comment by PaulHigin** [comment]
Good catch.  I actually wrote this a few years ago before information streams were added.  The code should be modified to handle them.

**Comment by PaulHigin** [comment]
Changes merged.  Thanks!

**Comment by PaulHigin** [comment]
Actually I had to revert this change because it causes .Net runtime errors on downlevel systems with v3 and v4 versions of PowerShell.  It looks like we would need to build different binaries to target different platform versions, and I prefer not to do that at least not yet.

**Comment by ajansveld** [comment]
This was fixed in #30.
